app/controllers/Api2.scala

```diff
@@ -68,6 +68,13 @@ class Api2 @Inject() (implicit val previewDataStore: PreviewDataStore,
     }
   }
 
+  def getPublishedMediaAtom(id: String) = APIHMACAuthAction {
+    publishedDataStore.getAtom(id) match {
+      case Some(atom) => Ok(Json.toJson(MediaAtom.fromThrift(atom)))
+      case None => Ok(Json.obj())
```

Comment:
I think so, there is nothing wrong if nothing gets returned, it just means that the atom has not been published. Happy to hear if you have a better suggestion though.